shared/middleware/helpers/redis_table.rb

```diff
@@ -10,6 +10,7 @@
 
 require_relative 'redis_property_bag'
 require 'securerandom' unless defined?(SecureRandom)
+require 'sinatra'
```

Comment:
According to the [docs](http://www.sinatrarb.com/intro.html#Sinatra::Base%20-%20Middleware,%20Libraries,%20and%20Modular%20Apps):

> Your file should require sinatra/base instead of sinatra; otherwise, all of Sinatra’s DSL methods are imported into the main namespace.